modules/gce_project/network.tf

```diff
@@ -62,23 +62,3 @@ resource "google_compute_firewall" "allow_ssh" {
     ports = [22]
   }
 }
-
-resource "google_compute_firewall" "allow_internal" {
-  name = "allow-internal"
-  network = "${google_compute_network.main.name}"
-  source_ranges = ["10.10.0.0/16"]
-
-  project = "${var.project}"
-
-  allow {
-    protocol = "icmp"
```

Comment:
Debugging, and there are several parts of IP that doesn't work very well without ICMP enabled (I don't remember them off the top of my head, but I'll see if I can dig up a link tomorrow).